components/team_up/sources/event-updated/event-updated.ts

```diff
@@ -0,0 +1,23 @@
+import { defineSource } from "@pipedream/types";
+import { DOCS_LINK } from "../../actions/list-events/list-events";
+import { EventTimestamps } from "../../common/responseSchemas";
+import common from "../common";
+
+export default defineSource({
+  ...common,
+  key: "team_up-event-updated",
+  name: "Event Updated",
+  description: `Emit new event when an event is updated [See docs here](${DOCS_LINK})`,
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
```

Comment:
```suggestion
```

It may happen for the same `id` to be updated twice in a short period of time

Reply:
You're right - I changed the way this worked later to the date filtering their API provides, and it no longer makes sense to dedupe on our side. Thanks!